static/js/apps/explore/app.tsx

```diff
@@ -330,6 +336,17 @@ const fetchFulfillData = async (
       classifications: classificationsJson,
       disableExploreMore,
     });
+    if (startTime) {
+      const endTime = window.performance
```

Comment:
nit: this wouldn't be end time right? this is the time taken?

Reply:
yes, elapsed time - updated.